atlp-rwanda/commanders-medica-backoffice
# Chore: configure deployment pipeline

#### What does this PR do?

This PR includes configurations for deployment of the web app.

#### Description of Task to be completed?

- Create a Dockerfile for the app
- Create a github action to run on every push to the `main` branch

#### How should this be manually tested?

After the action is complete check on [https://commanders-medica-backoffice.fly.dev](https://commanders-medica-backoffice.fly.dev) to check if new updates were added.

#### Any background context you want to provide?

Since fly provides a CLI, i deployed the app locally and made sure that it is deploying successfully. What is remaining is adding a `FLY_API_TOKEN` secret to this repo to make sure that the GitHub action is completing successfully

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A
